include/zephyr/bluetooth/hci.h

```diff
@@ -32,7 +32,11 @@ extern "C" {
  *
  * @return The string representation of the HCI error code.
  */
+#ifdef CONFIG_BT_HCI_ERR_TO_STR
```

Comment:
```suggestion
#if defined(CONFIG_BT_HCI_ERR_TO_STR)
```

Suggest to be consistent with the .c file